common/openapi.yml

```diff
@@ -26,6 +26,28 @@ paths:
           $ref: '#/components/responses/PermissionDenied'
         '500':
           $ref: '#/components/responses/ServerError'
+    delete:
+      summary: Unlink a user from an external account
+      tags: [ admin ]
+      operationId: adminDeleteLinkedAccount
+      requestBody:
+        content:
+          text/plain:
+            schema:
+              type: string
+              description: The Sam User Id for a user
```

Comment:
I'm unsure whether to have this be plain-text or a json object with a `userId` field. If you've got thoughts, lemme know!